content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -101,9 +100,58 @@ catch(Exception e)
 
 This sets three tags on the span: `"error.msg":exception.Message`,  `"error.stack":exception.ToString()`, and `"error.type":exception.GetType().ToString()`.
 
+## Adding spans
+
+If you aren’t using a supported framework instrumentation for [.NET Framework][6] or [.NET Core][7], or you would like additional depth in your application's [traces][8], you may want to add custom instrumentation to your code for complete flame graphs or to measure execution times for pieces of code.
+
+If modifying application code is not possible, use the environment variable `DD_TRACE_METHODS` to detail these methods.
+
+If you have existing `[Trace]` or similar attributes, or prefer to use attributes to complete any incomplete traces within Datadog, use Trace Annotations.
+
+### Datadog trace methods
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> This requires enabling automatic instrumentation for your application.
+</div>
+
+Using the `DD_TRACE_METHOD` environment variable, you can get visibility into unsupported frameworks without changing application code.
+
+```ini
+DD_TRACE_METHODS=Store.SessionManager[SaveSession]
+```
+
+The only difference between this approach and using `[Trace]` attributes is the customization options for the operation and resource names.  With DD Trace Methods, `operationName` is `trace.annotation` and `resourceName` is `SaveSession`.
```

Comment:
Didn't we change the resource names to be `SessionManager.SaveSession` here?

Reply:
Not yet...that's about to be merged in our repo so it's not in the publicly available Tracer yet